cmd/drone-server/drone.go

```diff
@@ -236,6 +244,14 @@ func main() {
 		auth.POST("", server.GetLogin)
 	}
 
+	redirects := r.Group("/redirect")
+	{
+		redirects.Use(server.SetDatastore(store))
+		redirects.Use(server.SetRepo())
+
+		redirects.GET("/:owner/:name/commit/:sha", server.RedirectSha)
```

Comment:
Jenkins it's a part Enterprise edition, i don't know, travis doesn't support gitlab